changebot/changelog.py

```diff
@@ -25,8 +25,12 @@ def find_prs_in_changelog_by_section(content):
     subcontent = ''
     previous = None
 
+    new_changelog_format = False
+    
     for line in content.splitlines():
-        if '-------' in line:
+        if '=======' in line:
+            new_changelog_format = True
+        if '=======' in line or (not new_changelog_format and '-------' in line):
```

Comment:
Not sure if I understand (it *is* past my bedtime...), but I don't see any difference logically between
```python
if '=======' in line or (not new_changelog_format and '-------' in line)
```
and
```python
if new_changelog_format or (not new_changelog_format and '-------' in line)
```

... anyway feel free to ignore if I misread your code.